docs/beast_priors.rst

```diff
@@ -262,39 +265,39 @@ The A(V) prior can be:
 
   av_prior_model = {'name': 'flat'}
 
-2. Lognormal with the maximum at the A(V) given by max_pos and the width
+2. Lognormal with the maximum at the A(V) given by mean and the width
 given by sigma.
 
 .. code-block:: python
 
   av_prior_model = {'name': 'lognormal',
```

Comment:
Single and double quotes were used in mix. Might want to use double quotes everywhere.

Reply:
Good point.  Done.